hack/kpng-local-up.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# build the kpng image...
+
+IMAGE="jayunit100/kpng-server:latest"
+
+function build {
+	cd ../
+
+	docker build -t $IMAGE ./
+	docker push $IMAGE
```

Comment:
isn't the push too much for a simple "local-up" ?

Reply:
Ya figure this will evolve over Time into a make file or somethin... do u prefer me to make a make file now ?

Reply:
I overlooked the issue, it's true that you need to push somewhere to deploy :s